gensim/test/test_keras_integration.py

```diff
@@ -0,0 +1,147 @@
+import unittest
+import os
+import sys
+import keras
+import numpy as np
+from gensim.models import word2vec
+from keras.engine import Input
+from keras.models import Model
+from keras.layers import merge
+from keras.preprocessing.text import Tokenizer
+from keras.preprocessing.sequence import pad_sequences
+from keras.utils.np_utils import to_categorical
+from keras.layers import Dense, Flatten
+from keras.layers import Conv1D, MaxPooling1D
+
+sentences = [
+    ['human', 'interface', 'computer'],
+    ['survey', 'user', 'computer', 'system', 'response', 'time'],
+    ['eps', 'user', 'interface', 'system'],
+    ['system', 'human', 'system', 'eps'],
+    ['user', 'response', 'time'],
+    ['trees'],
+    ['graph', 'trees'],
+    ['graph', 'minors', 'trees'],
+    ['graph', 'minors', 'survey']
+]
+
+module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
+datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
+
+
+class TestKerasWord2VecWrapper(unittest.TestCase):
+    def setUp(self):
+        self.model_cos_sim = word2vec.Word2Vec(sentences, size=100, min_count=1, hs=1)
```

Comment:
Done. Added a check to ensure that if Keras is not installed then the tests are skipped with a suitable error mesage.